### DISCUSSION TITLE
OpenSSL org source website is down?

### DISCUSSION THREAD

**Discussion by junaruga** [openssl/openssl#24984] [Q&A] [answered]
I am seeing the following error by downloading OpenSSL source archive files. I found the issue on [our CI](https://github.com/junaruga/ruby-openssl/actions/runs/10075912441/job/27855166257). What's happening on your website now?

```
$ curl -OL https://openssl.org/source/openssl-1.0.2u.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   127  100   127    0     0   1182      0 --:--:-- --:--:-- --:--:--  1186

$ cat openssl-1.0.2u.tar.gz
<?xml version='1.0' encoding='UTF-8'?><Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message></Error>
```

**Comment by nhorman** [comment]
Appears we are missing a redirect for our sources.  Until we get that fixed up, you can find those sources here:
https://openssl-library.org/source/old/1.0.2/index.html

**Reply by junaruga** [reply]
All right. Thanks for telling me the workaround. I hope the website issue will be fixed soon.

**Reply by t-j-h** [reply]
> All right. Thanks for telling me the workaround. I hope the website issue will be fixed soon.

It will be fixed shortly ...

**Reply by junaruga** [reply]
Note we applied a temporary workaround https://github.com/ruby/openssl/pull/779 for this issue.

**Comment by t-j-h** [comment, marked as answer]
This should be resolved. Please check.

**Reply by junaruga** [reply]
Yes, I checked it. Now the original downloading links on the website work!
